src/structures/Guild.js

```diff
@@ -921,7 +921,7 @@ class Guild extends Base {
    * @returns {Promise<GuildChannel>}
    * @example
    * // Create a new text channel
-   * guild.createChannel('new-general', 'text')
+   * guild.createChannel('new-general', {type: 'text'})
```

Comment:
Since `text` is the default value for `type`, I think it's redundant to include that in here. Maybe use `reason` instead?

Reply:
I realize it's pointless as text, but maybe swap it to `voice` then? Or could just set do reason. The goal was to show "Type goes here if you want it".

Will lint after decision is made.